piet-coregraphics/src/text.rs

```diff
@@ -223,6 +228,9 @@ impl CoreGraphicsTextLayoutBuilder {
 
         // 'wght' as an int
         const WEIGHT_AXIS_ID: i32 = 2003265652;
+        // taken from android:
+        // https://api.skia.org/classSkFont.html#aa85258b584e9c693d54a8624e0fe1a15
+        const SLANT_RADIANS: f64 = 0.25;
```

Comment:
I would name this "SLANT_TANGENT", as it's the tan of the actual angle (~0.245 radians = ~14°). It's a *very* small difference, but I think being precise can avoid confusion.